pom.xml

```diff
@@ -63,6 +65,16 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Does that work now? I appreciate the effort to make it easier to run tests more often (with CI/CD ideal) but consider also simple alternative like a recipe on "how to run tests before merging" if the plugins are making trouble. There's only so much time one can spend on such things.

Reply:
It works just for testing pure JavaScript functions. It does not work if you want your JavaScript to interact with an html page, so you need to test that part separately.